docker/create_weekly_tables.py

```diff
@@ -54,18 +54,43 @@ def get_connection():
     return psycopg2.connect(dsn)
 
 
+def get_raw_crash_tables(cursor):
+    existing = set()
+    cursor.execute(
+        'SELECT table_name FROM information_schema.tables WHERE table_name LIKE \'raw_crash%\''
+    )
+    for row in cursor.fetchall():
+        existing.add(row[0])
+    return existing
+
+
 def main(args):
     today = datetime.datetime.utcnow()
     eight_weeks_ago = today - datetime.timedelta(days=(7 * 8))
 
     conn = get_connection()
     cursor = conn.cursor()
 
+    # Query raw_crashes tables
+    existing = get_raw_crash_tables(cursor)
+
     # This starts 8 weeks ago and tells the procedure to make 10 weeks worth of tables. So that
     # covers 8 weeks ago up to 2 weeks from now. That should be sufficient for any kinds of
     # debugging we're doing.
     cursor.callproc('weekly_report_partitions', [10, eight_weeks_ago.strftime('%Y-%m-%d')])
 
+    # Query tables again and print out new ones
+    all_tables = get_raw_crash_tables(cursor)
+    new_tables = all_tables - existing
+    if new_tables:
+        for table_name in new_tables:
+            print('Created %s' % table_name)
+    else:
+        print('Created no new tables')
+
+    conn.commit()
```

Comment:
Pretty sure the lack of this commit is the cause of https://bugzilla.mozilla.org/show_bug.cgi?id=1394821 .